File: backend/slotify/spring/src/main/java/com/slotify/backend/spring/auth/dtos/RegisterCompanyResponse.java
package com.slotify.backend.spring.auth.dtos;

import lombok.Builder;
import lombok.Getter;
import lombok.ToString;

import java.util.UUID;

@Builder
@Getter
@ToString
public class RegisterCompanyResponse {

    private UUID userId;
    private String username;
    private String email;
    private String physicalAddress;
}

File: app/src/main/java/com/raj/slotify/dtos/registry/company/RegisterCompanyResponse.kt
package com.raj.slotify.dtos.registry.company

import com.google.gson.annotations.SerializedName
import java.util.UUID

data class RegisterCompanyResponse(

    @SerializedName("userId")
    val userId: UUID,

    @SerializedName("username")
    val username: String,

    @SerializedName("email")
    val email: String,

    @SerializedName("physicalAddress")
    val physicalAddress: String

)